elated-egrets/bangazon-internal-site
# created list view for the employee model

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout JRemployee_list
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. run the server
1. make sure the Employee url pull up with dummy page

## What you changed
added the list view for the employee model
## What was added to read me
NA
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
